pages/projects/affiliated.md

```diff
@@ -2,7 +2,8 @@
 layout: page
 title: Affiliated packages
 permalink: /affiliated
-nav_order: 5
+nav_order: 1
```

Comment:
Are the affiliated projects coming prior to the project packages themselves? I may be misinterpreting the 1 here.

Reply:
These are now nested. See the screenshot above (not the phone one).

Reply:
I see.